Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'jbuilder', '~> 2.0' # Build JSON APIs with ease. Read more: https://github.
 gem 'jquery-rails' # Use jquery as the JavaScript library
 gem 'jquery-turbolinks'
 gem 'jquery-ui-rails'
+gem 'mini_racer'
 gem 'mysql2'
```

Comment:
Switch from therubyracer to mini_racer as the former depends on an early version of V8 which I couldn't get to compile on Catalina.